### DISCUSSION TITLE
Automatic Deployments Not Working

### DISCUSSION THREAD

**Discussion by JulzPolz** [vercel/community#7344] [Account Issue]
### Question

Hi,

I have a project which isn't running new deployments when i push to GIT (on any branch, with my user). I have another project, which is working perfectly.

Both project configurations are the same. Please let me know how i can fix it, as without it, it breaks a lot of vercel's CI functionality that i am interested in.

Thanks.

**Comment by pawlean** [comment]
Hi @JulzPolz!

Let's try and debug this together. 😃  Can you check if you've tried these?

- Go to the Vercel dashboard for the project and check the **Deployments** tab. Look for any error messages or warnings that might indicate what's preventing the deployment.
- Ensure that your **Git repository** is correctly connected to Vercel. You can do this by going to the Settings tab in the Vercel dashboard, then clicking on Git. Check that the repository URL, branch, and credentials are correct.
- Go to the **Logs** tab in the Vercel dashboard and filter the logs by Deployment. Look for any error messages or warnings that might indicate what's preventing the deployment.
- Go to the **Settings** tab in the Vercel dashboard, then click on **Build & Deployment**. Ensure that the build command, output directory, and deployment settings are correct.
- Go to the **Deployments** tab in the Vercel dashboard and click the **Deploy** button. Select the correct branch and commit hash. If the deployment fails, you'll see an error message that might help you identify the issue.

[a comment by an automated account is not shown]
